pkg/steps/release/import_release.go

```diff
@@ -67,7 +66,8 @@ type importReleaseStep struct {
 }
 
 func (s *importReleaseStep) Inputs() (api.InputDefinition, error) {
-	return api.InputDefinition{s.pullSpec}, nil
+	input, err := s.source.Input(context.Background())
```

Comment:
I'm not sure if just this single case is enough to justify a change of [this signature](https://github.com/openshift/ci-tools/blob/56fd6a6664337b8d989ea5940387abc7f08f5bfe/pkg/api/graph.go#LL22C2-L22C10) into:
```golang
Inputs(ctx context.Context) (InputDefinition, error)
````

Reply:
https://github.com/openshift/ci-tools/pull/483/commits/aac2db8bbbdd5af929334c1c15d00feaabc6558c

I considered making this change simply as a matter of performance (to avoid creating useless `Context` objects), but it didn't seem like enough justification